halo.go

```diff
@@ -1,14 +1,2 @@
 package halo
 
-import (
-	"github.com/cmwaters/halo/consensus"
-	"github.com/cmwaters/halo/network"
-	"github.com/cmwaters/halo/pkg/sign"
-
-	"github.com/libp2p/go-libp2p/core/host"
-)
-
-func New(host host.Host, signer sign.Signer, parameters consensus.Parameters) *consensus.Engine {
-	gossipLayer := network.NewLibP2PGossip(host)
-	return consensus.New(gossipLayer, signer, parameters)
-}
```

Comment:
I am removing this for now to avoid the burden. It's not useful, and constructors will change often in this and further PRs

Reply:
Okay, this was more for me to keep in mind how we expect users to use this library